### PR TITLE
Added Avocado

### PR DESCRIPTION
**Pull request by ishanjain28**
Hey, This PR is for adding Instadapp's Avocado product on this list.

* https://avocado.instadapp.io
* https://instadapp.io

### CODE REVIEW

File: _data/chains/eip155-634.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "infoURL": "https://avocado.instadapp.io",
+  "shortName": "avocado",
+  "chainId": 634,
+  "networkId": 634,
+  "icon": "avocado",
+  "explorers": [
+    {
+      "name": "avoscan",
+      "url": "https://avoscan.co/",
```
Comment (ligi) [inline _data/chains/eip155-634.json:19]:
```suggestion
      "url": "https://avoscan.co",
```

**Review by ligi — APPROVED** [review]

File: _data/chains/eip155-634.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      "icon": "avocado",
+      "standard": "EIP3091"
```
Comment (ligi) [inline _data/chains/eip155-634.json:21]: explorer does not seem to be EIP3091 compatible 
```suggestion
      "standard": "none"
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by ishanjain28** [comment]
Hey, There was an issue with the IPFS node and it has been fixed. Please re-run the pipeline and it should be able to fetch the assets now.

**Comment by ishanjain28** [comment]
Hey @ligi Please review this again

**Comment by ishanjain28** [comment]
Hi @ligi, I hope you are doing well. I have rebased the PR on latest head in `master`.

Please review this PR and approve gh action runs.

**Comment by ishanjain28** [comment]
Hi, Thank you for re-running gh actions for this PR and I am sorry about this failing again. 

I didn't realize ipfs had bit of a problem related to memory consumption and it was repeatedly getting killed by oomkiller. I have worked around that problem and it should stay online from now on. Please re-run the pipeline, Thank you!

**Review by alokeutpal — APPROVED** [review]

**Pull request merged**